checker.py

```diff
@@ -227,9 +225,27 @@ def git_checker(
         if not check_clean(lines):
             # WD dirty
             dirty_dirs += [gd]
-        if check_unpushed(lines):
-            # Changes unpushed
-            unpushed_dirs += [gd]
+
+    # Then, check for unpushed branches with a configured remote. This is done by using
+    # git log remote/branch..branch.
+    unpushed_branches: List[str] = []
+    for gd in itr:
```

Comment:
Do you think we can keep this under the same loop (`for gd in itr`) above? That way we won't need two progress bars to run this.

Reply:
With ac2b98c, I pulled the two loops into one loop.